### DISCUSSION TITLE
feature: experience penalty when killing weaker enemies

### DISCUSSION THREAD

**Discussion by ghost** [AscensionGameDev/Intersect-Engine#2210] [Ideas]
**Is your feature request related to a problem? Please describe.**
This is very good for balancing the UP and farm system in MMORPGs, if a player farms a low level monster he should receive a lower amount of XP, if possible configurable.
As with Ragnarok Online,

**Describe the solution you'd like**
if you kill a monster with up to 15 levels more than yours, you can earn 50% to 75% more XP, and if you kill monsters with 10 levels less, you will receive a 50% penalty there is 75% XP of monsters.

**Comment by WeylonSantana** [comment]
@ThousandX I'm sorry to intrude on your feature request, but, why don't you do it for events? It's possible.

**Comment by Cheshire92** [comment]
Going through events is great and all, but a very user unfriendly way of doing this.

This should be doable, but would need to be configurable properly. (Ie setting the drop-off levels in a config file as well as how much exp is taken away)

**Comment by pandinocoder** [comment]
Restored issue.

**Comment by dhmello** [comment]
This was implemented in my game, but I used a metric of up to 15 levels of difference between the player and/or party member, which will gain 0 xp if the difference between the player and the monster is 15 levels or more.
